### ISSUE TITLE
anita downloads "blank" pics

### ISSUE THREAD

**Issue by Leryn** [SachsenCoder/Anita#26]
i think the site is changed something to the pics, the direct link to every pics, because anita downloads the pics but are all "blank", the pics are empty and i can't see anything

**Comment by ChristianG1984** [comment]
I had the same issue some minutes ago when I tried to download some pics. It's a strange problem, because when I paste the link directly in the browser, the picture is loaded flawlessly. :-/

**EDIT**: No, it does not load in the browser. It was just in the cache, when I tried it out. This time, the problem is much harder to solve. I'm searching for a solution.

**Comment by ChristianG1984** [comment]
**I just solved the problem!!**
It's actually downloading pictures on my system! I will update the App soon. :-)

**Comment by Leryn** [comment]
thank you very much, you're an hero mate!   anyway i'm not sure was a problem of your app, but of the site. for example here

https://pics.wikifeet.com/Jennifer-Lawrence-Feet-1515888.jpg

direct link of a jennifer's pic but i can't save it

**Comment by ChristianG1984** [comment]
@Leryn They changed a massively big thing on their site. They use a new viewer engine. So, the example-link does not point to the raw image anymore. It is redirected in some strange way. (Well, at this moment, I can access the picture directly. Maybe, they are actually within the changing process or so.)

**Comment by Leryn** [comment]
yes, now i can download the pics with the app btw.. weird thing. thx for help

**Comment by Leryn** [comment]
the site is changed again, and again anita can't download any pics not only from wikifeetX but also from the main site wikifeet. please fix this if u can, thank you

**Comment by ChristianG1984** [comment]
OK, I will take care of that soon.

**Comment by Leryn** [comment]
> OK, I will take care of that soon.


please man, we need it. i'm begging you, fix the software
